Leanplum-SDK/Classes/LPActionContext-Internal.h

```diff
@@ -21,28 +21,22 @@
                          originalMessageId:(NSString *)originalMessageId
                                   priority:(NSNumber *)priority;
 
-@property (readonly, strong) NSString *private_Name;
-@property (readonly, strong) NSString *private_MessageId;
-@property (readonly, strong) NSString *private_OriginalMessageId;
-@property (readonly, strong) NSNumber *private_Priority;
-@property (readonly, strong) NSDictionary *private_Args;
-@property (readonly, strong) LPActionContext *private_ParentContext;
-@property (readonly) int private_ContentVersion;
-@property (readonly, strong) NSString *private_Key;
-@property (readonly) BOOL private_PreventRealtimeUpdating;
-@property (readonly) BOOL private_IsRooted;
-@property (readonly) BOOL private_IsPreview;
+@property (readonly, strong) NSString *name;
+@property (readonly, strong) NSString *messageId;
+@property (readonly, strong) NSString *originalMessageId;
+@property (readonly, strong) NSNumber *priority;
+@property (readonly, strong) NSDictionary *args;
+@property (readonly, strong) LPActionContext *parentContext;
+@property (readonly) int contentVersion;
+@property (readonly, strong) NSString *key;
+@property (readonly) BOOL preventRealtimeUpdating;
+@property (nonatomic, assign) BOOL isRooted;
+@property (nonatomic, assign) BOOL isPreview;
```

Comment:
Ah.. this is intentional because there were separate methods to set those values also, which could just be collapsed into the property since they weren't being readonly

Reply:
![image](https://user-images.githubusercontent.com/761132/43304959-e1a1e938-912a-11e8-8b1e-e7782e566084.png)